code/api/src/modules/delivery/resolvers.js

```diff
@@ -11,6 +11,25 @@ export async function get(parentValue, { id }) {
   })
 }
```

Comment:
Apparently there's a merge conflict with main somewhere in this file? Not sure where though, this looks right to me